specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -2263,9 +2263,13 @@
           "$ref": "../datafactory.json#/definitions/SecretBase"
         },
         "securityToken": {
-          "description": "The security token is required to remotely access Salesforce instance.",
+          "description": "The security token is optional to remotely access Salesforce instance.",
           "$ref": "../datafactory.json#/definitions/SecretBase"
         },
+        "apiVersion": {
+          "type": "object",
```

Comment:
should it be string?

Reply:
@MyronFanQiu If we set object type, customer can input a expression instead a string value.

Reply:
what's the meaning of expression? Could you give me an example? I think an expression is also a string.